clang/lib/Sema/SemaTemplateInstantiateDecl.cpp

```diff
@@ -4856,7 +4856,7 @@ bool TemplateDeclInstantiator::SubstDefaultedFunction(FunctionDecl *New,
                    : DFI);
   }
 
-  SemaRef.SetDeclDefaulted(New, Tmpl->getLocation());
+  SemaRef.SetDeclDefaulted(nullptr, New, Tmpl->getLocation());
```

Comment:
I did wonder if my new test (https://godbolt.org/z/hE3h4b4oM) with class template would fail on `nullptr`s in `SemaTemplateInstantiateDecl.cpp`, but I guess it doesn't. Could there be a scope that we should pass here, too?
CC @cor3ntin @mizvekov

Reply:
@Endilll 
I'm not sure if I miss some steps.
Do I need to update anything to make this PR available?